apps/src/sites/studio/pages/levels/_dialog.js

```diff
@@ -8,63 +8,69 @@ import { UnsubmitDialog } from '@cdo/apps/lib/ui/LegacyDialogContents';
  * This file does some handling of submit button interactions.
  */
 
-// Are we read-only?  This can be because we're a teacher OR because an answer
-// has been previously submitted.
-if (appOptions.readonlyWorkspace) {
-  // hide the Submit button.
-  $('.submitButton').hide();
+// It's worth noting that in levelgroups containing external type levels, this
+// JS can get loaded more than once. It's important that this code runs after
+// the DOM is done loading, otherwise we might not actually have all of our
+// buttons yet
+$(document).ready(() => {
```

Comment:
Change to this file are probably most easily viewed with whitespace diffs ignored https://github.com/code-dot-org/code-dot-org/pull/19753/files as it's mostly wrapping things in the document.ready